src/pixie.nim

```diff
@@ -351,17 +359,25 @@ proc strokeText*(
   text: string,
   color: SomeColor,
   transform: Vec2 | Mat3 = vec2(0, 0),
-  strokeWidth = 1.0
+  strokeWidth = 1.0,
+  bounds = vec2(0, 0)
```

Comment:
I fear a little with default bounds 0,0 if you are not doing it in code and as you shrink the text might explode out.